server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileService.java

```diff
@@ -57,7 +59,7 @@
   private final QProfileExporters exporters;
 
   public QProfileService(DbClient db, ActivityIndex activityIndex, RuleActivator ruleActivator, QProfileFactory factory,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)Method has 8 parameters, which is greater than 7 authorized.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)